src/common/normalize.ts

```diff
@@ -96,3 +68,34 @@ export function normalizeEvent(event: Event): Event {
   delete tags.server_name;
   return event;
 }
+
+/**
+ * Returns a reference to the exception stack trace in the given event.
+ * @param event An event potentially containing stack traces.
+ */
+function getStacktrace(event: Event): Stacktrace | undefined {
+  const { stacktrace, exception } = event;
+
+  // Try the main event stack trace first
+  if (stacktrace) {
+    return stacktrace;
+  }
+
+  if (exception) {
+    // Raven Node adheres to the Event interface
+    // @ts-ignore: need to be able to index exception
+    if (exception[0]) {
+      // @ts-ignore: need to be able to index exception
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      return exception[0].stacktrace;
+    }
+
+    // Raven JS uses the full values interface, which has been removed
+    const raven = exception as any as { values: Exception[] };
+    if (raven.values && raven.values[0]) {
+      return raven.values[0].stacktrace;
```

Comment:
Yeah, this can probably be heavily refactored. I've just copied this over from the [existing code](https://github.com/getsentry/sentry-electron/pull/361/files/0480316d77d8ab472a4353aee9fb9d2c847be36a#diff-f60f8780fa9e3d6815702a4e03640e02b2be051b7fbf653cae03c3eae2d6eae8L35-L60) because it's always been there and it was probably copied from somewhere else before that!